test/os-queens-environment.json

```diff
@@ -0,0 +1,169 @@
+{
+	"id": "8e0a9736-c3eb-452e-ae9d-64fd8e94cb0c",
```

Comment:
1. Maybe we need to remove the generated environments: RULE_ID SERVICE_ID ….
2. Suggest that those generated environments be named with lower-case in collection.json so that we can distinguish them with  those given outside.

Reply:
When we run this test in postman, those var will be add to environment json again, and will be exported. So that everyone need to remove them before check-in file. I think we may not let other to take time to do that.

If all vars are in this json, the ones who do have intial value always require a initial value.